Sources/ApiVideoPlayer/ApiVideoPlayerInformationNowPlaying.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+import MediaPlayer
+
+protocol InformationNowPlaying {
+    var nowPlayingData: NowPlayingData? { get set }
+    func pause(currentTime: CMTime)
+    func play(currentTime: CMTime)
+    func updateCurrentTime(currentTime: CMTime)
+    func updatePlaybackRate(rate: Float)
+}
+
+class ApiVideoPlayerInformationNowPlaying: InformationNowPlaying {
+    private var infos = [String: Any]()
+    private let taskExecutor: TasksExecutorProtocol.Type
+
+    init(taskExecutor: TasksExecutorProtocol.Type) {
+        self.taskExecutor = taskExecutor
+    }
+
+    var nowPlayingData: NowPlayingData? {
+        didSet {
+            guard let nowPlayingData = nowPlayingData else {
+                MPNowPlayingInfoCenter.default().nowPlayingInfo = [:]
+                return
+            }
+            if let title = nowPlayingData.title {
+                infos[MPMediaItemPropertyTitle] = title
+            }
+            infos[MPMediaItemPropertyPlaybackDuration] = nowPlayingData.duration.seconds
+            infos[MPNowPlayingInfoPropertyElapsedPlaybackTime] = nowPlayingData.currentTime.seconds
+            infos[MPNowPlayingInfoPropertyIsLiveStream] = nowPlayingData.isLive
+
+            infos[MPNowPlayingInfoPropertyPlaybackRate] = 1.0
```

Comment:
hardcoding playbackrate to 1.0 here is not a great idea.
Could you add `playbackRate` in `NowPlayingData`?